src/main/webapp/app/shared/image/secured-image.component.ts

```diff
@@ -63,20 +73,37 @@ export class SecuredImageComponent implements OnChanges {
     }
 
     private loadImage(url: string): Observable<any> {
-        return this.httpClient
-            .get(url, { responseType: 'blob' })
-            .map(e => {
+        return of(null).pipe(
```

Comment:
some more comments would be really helpful here!

Reply:
You are right, this should have had JSDoc, I added some and also more comments in the method.